### PR TITLE
DOCTEAM-1338:Setting up a systemd service-article

### PR DESCRIPTION
**Pull request by Amrita42**
### Description

The scope of this PR is to cover the scope of "Setting up a systemd service" as defined in https://jira.suse.com/browse/DOCTEAM-1338

**Reviewers**
Please dont review now , will tag in slack when ready.
 
**Reused** (Review not required as this topics have been reviewed and are being re-used multiple times)
- What is systemd?
- Structure of a unit file
- Unit file types
- Unit dependencies and order

**Review required**
- Creating a Linux service with systemd
- Editing a systemd unit file
- Debugging a systemd service

### CODE REVIEW

**Comment by Amrita42** [comment]
[setting-up-systemd_en.pdf](https://github.com/user-attachments/files/16409116/setting-up-systemd_en.pdf)

**Review by taroth21 — CHANGES_REQUESTED** [review]
@Amrita42 : Many thanks, well done! Sorry for the large number of comments and suggestions but the majority of them is about updating the metadata and adjusting some structures to the latest template updates (I provide those as suggestions). 

Regarding the content, only some minor nitpicks. Many thanks!The majority of the changes I suggested (as proposals) refer to metadata and adjustments to comply with the latest templates

File: references/systemd-editing-unit-file.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+<!-- refer to README.md for file and id naming conventions -->
+<!-- metadata is dealt with on the assembly level -->
+<topic xml:id="systemctl-edit-service-file"
+ role="reference" xml:lang="en"
+ xmlns="http://docbook.org/ns/docbook" version="5.2"
+ xmlns:its="http://www.w3.org/2005/11/its"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xmlns:trans="http://docbook.org/ns/transclusion">
+  <info>
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply references/systemd-editing-unit-file.xml:18]: I will use the second option and rephrase the title

File: references/systemd-debug-failed-service.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      <para>
+Use the <command>systemctl</command> and <command>journalctl</command> commands to investigate the reasons for a failed system service.
+        </para>
+    </abstract>
+  </info>
+      <para>When &systemd; fails to start a service, a generic error message is displayed, for example:</para>
+<screen>&prompt.root;<command>systemctl</command> start apache2
+Job for apache2.service failed because the control process exited with error code.
+See "systemctl status apache2.service" and "journalctl -xe" for details.</screen>
+        <itemizedlist>
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply references/systemd-debug-failed-service.xml:31]: Yes , good point I agree +1

File: references/systemd-editing-unit-file.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Amrita42) [reply references/systemd-editing-unit-file.xml:18]: Change of name will then need to reflect in other three published articles:
systemd basics
managing systemd services
systemd boot process

This topic is used in the above three .

should i go ahead ? @taroth21

File: tasks/systemd-edit-service.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+<!DOCTYPE topic
+[
+  <!ENTITY % entities SYSTEM "../common/generic-entities.ent">
+    %entities;
+]>
+<!-- refers to legacy doc: https://github.com/SUSE/doc-sle/blob/main/xml/adm_sudo.xml -->
+<!-- point back to this document with a similar comment added to your legacy doc piece -->
+<!-- refer to README.md for file and id naming conventions -->
+<!-- metadata is dealt with on the assembly level -->
+<topic xml:id="edit-systemd-service"
```
Comment (Amrita42) [inline tasks/systemd-edit-service.xml:11]: @taroth21 Looking at this more in-depth , this file is not required and I will remove it , as we do have an existing one "systemd-editing-unit-file" in place .

File: references/systemd-editing-unit-file.xml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      </para>
+    </abstract>
+  </info>
+  <para>There are three main directories where unit files are stored on the system: </para>
+  <variablelist>
+    <varlistentry>
+      <term><filename>/usr/lib/systemd/system/</filename></term>
+      <listitem>
+        <para>
+          When the package is installed, &systemd; unit files reside here.
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply references/systemd-editing-unit-file.xml:34]: Yes , the systemd package , rephrased

[the remark this answers is not in the document]
Comment (Amrita42) [reply references/systemd-editing-unit-file.xml:34]: its the systemd unit files with RPM packages , re-phrased.

File: references/systemd-editing-unit-file.xml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    <!--add author's email address-->
+    <meta name="maintainer" content="amrita.sakthivel@suse.com"/>
+    <abstract>
+      <para>
+        You can use the <command>systemctl</command> command to edit and modify an existing service file.
+      </para>
+    </abstract>
+  </info>
+  <para>There are three main directories where unit files are stored on the system: </para>
+  <variablelist>
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply references/systemd-editing-unit-file.xml:29]: Done , defined which directory takes precedence

File: tasks/systemd-edit-service.xml (hunk 3 above)
Comment (Amrita42) [reply tasks/systemd-edit-service.xml]: Ignore

File: references/systemd-editing-unit-file.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Amrita42) [reply references/systemd-editing-unit-file.xml:18]: Resolved in the chat , ignore

**Pull request closed without merging**